bundles/org.eclipse.ui.views.log/src/org/eclipse/ui/internal/views/log/messages.properties

```diff
@@ -47,7 +47,9 @@ LogView_confirmOverwrite_message = File " {0}" exists. Would you like to overwri
 LogView_confirmDelete_deleteButton = &Delete All Events
 LogView_operation_importing = Importing log...
 LogView_operation_reloading = Reloading...
-LogView_activate = &Activate on new events
+LogView_activate = &Activate on any new event
+LogView_activateWarn = Activate on new warnings
+LogView_activateError = Activate on new error only
```

Comment:
I like the proposal.
Instead of making the user assume that probably "activate on new warnings" also includes activation on error, I would make that explicit.
Nitpicky: it should at least be "warning" instead of "warnings" to be consistent with the the other entries.
```suggestion
LogView_activateWarn = Activate on new warning or error
LogView_activateError = Activate on new error
```

Reply:
ok

Reply:
> I like the proposal. Instead of making the user assume that probably "activate on new warnings" also includes activation on error, I would make that explicit. Nitpicky: it should at least be "warning" instead of "warnings" to be consistent with the the other entries.

This also fits more to the filter options of the problems view:
<img width="361" alt="image" src="https://github.com/eclipse-platform/eclipse.platform.ui/assets/28338612/b4fff307-a5be-4a6a-a451-258f1d484f64">
where it's also made explicit.